dummyfs/Makefile

```diff
@@ -4,8 +4,14 @@
 # Copyright 2017, 2018 Phoenix Systems
 #
 
-NAME := dummyfs
 LOCAL_PATH := $(call my-dir)
-SRCS := $(wildcard $(LOCAL_PATH)*.c)
+DUMMYFS_SRCS := $(wildcard $(LOCAL_PATH)*.c)
 
+NAME := dummyfs
+SRCS := $(DUMMYFS_SRCS)
```

Comment:
not doing full review, just something I caught in mail notification - You should use built library to link the binary, don't reuse .o files.